Livecheckables/aspell.rb

```diff
@@ -0,0 +1,4 @@
+class Aspell
+  livecheck :url   => "https://ftp.gnu.org/gnu/aspell/",
```

Comment:
```suggestion
  livecheck :url   => "https://ftp.gnu.org/gnu/aspell/?C=M&O=D",
```

With index pages, we can sometimes append the `?C=M&O=D` query string parameters to sort the items on the page by their last modified date in descending order. This doesn't impact livecheck's ability to identify the newest version but it's nice to have them in order if we ever check the debugging output (e.g., `brew livecheck --debug aspell`).